openmls/src/group/mls_group/mod.rs

```diff
@@ -418,28 +977,46 @@ impl MlsGroup {
                     plaintext.set_membership_tag(
                         provider.crypto(),
                         self.ciphersuite(),
-                        self.group.message_secrets().membership_key(),
-                        self.group.message_secrets().serialized_context(),
+                        self.message_secrets().membership_key(),
+                        self.message_secrets().serialized_context(),
                     )?;
                 }
                 plaintext.into()
             }
             OutgoingWireFormatPolicy::AlwaysCiphertext => {
                 let ciphertext = self
-                    .group
-                    .encrypt(
-                        mls_auth_content,
-                        self.configuration().padding_size(),
-                        provider,
-                    )
+                    .encrypt(mls_auth_content, provider)
                     // We can be sure the encryption will work because the plaintext was created by us
                     .map_err(|_| LibraryError::custom("Malformed plaintext"))?;
-                MlsMessageOut::from_private_message(ciphertext, self.group.version())
+                MlsMessageOut::from_private_message(ciphertext, self.version())
             }
         };
         Ok(msg)
     }
 
+    // Encrypt an PublicMessage into an PrivateMessage
```

Comment:
The input is not a piblic message, the wording is misleading.

Reply:
As far as I remember this is just copied from elsewhere. But I'll fix it.